build.gradle

```diff
@@ -23,6 +23,7 @@ plugins {
 allprojects {
     plugins.withType(AndroidBasePlugin).configureEach {
         android {
+            buildToolsVersion "35.0.1"
```

Comment:
You only need to keep this. BTY, can we just move `compileSdk` to 36?

Reply:
@Goooler I just double-check, apparently, Gradle will only one build tool, and we can use build tool 36.0.0.

Will update this PR in a moment.

---

If you'd like to know why: my local Gradle configuration is messed up so that's why Gradle is using more than 1 build tool, and during Android 16 r2 merge to Lawnchair, some of the files were in Kotlin and because the Kotlin compiler is more strict than java compiler, many of the classes from framework.jar is left unresolved.

Reply:
I believe this PR is outdated if the build tool 36.0.0 works for you, as it defaults to compileSdk to 36.

Reply:
> I believe this PR is outdated if the build tool 36.0.0 works for you, as it defaults to `compileSdk = 36`.

Whoops 😅, I believe your right. I somehow forgot that you have bumped the sdk compilation to 36.